README.md

```diff
@@ -1,6 +1,6 @@
 [![nbirillo](https://circleci.com/gh/nbirillo/ast-transformations/tree/master.svg?style=shield)](https://app.circleci.com/pipelines/github/nbirillo/ast-transformations?branch=master)
 
-# Ast-transformations
+# AST-transformations
 
 An IntelliJ-based IDE plugin for Python AST transformations
```

Comment:
maybe change to "An IntelliJ-based IDE plugin for performing Python AST transformations using PSI elements"?

Reply:
Great idea, thanks!